Content.Client/Chat/UI/SpeechBubble.cs

```diff
@@ -251,6 +253,8 @@ protected override Control BuildBubble(ChatMessage message, string speechStyleCl
                     speechStyleClass = message.SpeechStyleClass;
                 else if (entityManager.HasComponent<SquadLeaderComponent>(senderUid) || entityManager.HasComponent<HiveLeaderComponent>(senderUid))
                     speechStyleClass = "commanderSpeech";
+                else if (entityManager.HasComponent<XenoWordQueenComponent>(senderUid) || entityManager.HasComponent<InnateCommandSpeechComponent>(senderUid))
+                    speechStyleClass = "commanderSpeech";
```

Comment:
Remove the check for `XenoWordQueenComponent` and just add `InnateCommandSpeechComponent` to the queen prototype. I think it would be redundant, and this form of checking unrelated components is bad form.
```suggestion
                else if (entityManager.HasComponent<SquadLeaderComponent>(senderUid) || entityManager.HasComponent<HiveLeaderComponent>(senderUid) || entityManager.HasComponent<InnateCommandSpeechComponent>(senderUid))
                    speechStyleClass = "commanderSpeech";
```